rsmtool/preprocessor.py

```diff
@@ -2463,13 +2475,31 @@ def process_data_rsmpredict(self, config_obj, data_container_obj):
         h1_mean = df_postproc_params['h1_mean'].values[0]
         h1_sd = df_postproc_params['h1_sd'].values[0]
 
+        #if the we are using a newly trained model, use trim_tolerance from the
+        # df_postproc_params. If not, set it to the default value and show
+        # deprecation warning
+        if 'trim_tolerance' in df_postproc_params.columns:
+            trim_tolerance = df_postproc_params['trim_tolerance'].values[0]
+        else:
+            trim_tolerance = 0.49998
+            logging.warning("The tolerance for trimming scores will be assumed to be 0.49998, "
+                            "the default value in previous versions of RSMTool. "
+                            "We recommend re-training the model to ensure future "
+                            "compatibility.")
+
+
         # now generate the predictions for the features using this model
         logged_str = 'Generating predictions'
         logged_str += ' (expected scores).' if predict_expected_scores else '.'
         logging.info(logged_str)
+
+        # compute minimum and maximum score for expected predictions
+        min_score = int(np.rint(trim_min - trim_tolerance))
```

Comment:
If you're doing `np.rint()`, do you need to cast these to `int()` afterward?

Reply:
Surprisingly, yes: 

```In [2]: a = np.rint(5.4)

In [3]: a
Out[3]: 5.0

In [4]: type(a)
Out[4]: numpy.float64

In [5]: type(int(a))
Out[5]: int```